data/simple-icons.json

```
@@ -6042,6 +6042,11 @@
 		"source": "https://www.ghostery.com",
 		"guidelines": "https://www.ghostery.com/press/"
 	},
+	{
+		"title": "Ghostfolio",
+		"hex": "36CFCC",
```

```suggestion
		"hex": "36CFCB",
```

Looks like `#36CFCB` is correct color.